bin/cluster

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+import argparse
+import ConfigParser
+import sys
+import os
+
+
+class Cluster(object):
+    """Python wrapper to ensure environment is correct for running ansible playbooks
+    """
+
+    def __init__(self, args):
+        self.args = args
+
+        # setup ansible ssh environment
+        if 'ANSIBLE_SSH_ARGS' not in os.environ:
+            os.environ['ANSIBLE_SSH_ARGS'] = (
+                '-o ForwardAgent=yes'
+                ' -o StrictHostKeyChecking=no'
+                ' -o UserKnownHostsFile=/dev/null'
+                ' -o ControlMaster=auto'
+                ' -o ControlPersist=600s'
+            )
+
+    def apply(self):
+        # setup ansible playbook environment
+        config = ConfigParser.ConfigParser()
+        if 'gce' == self.args.provider:
+            config.readfp(open('inventory/gce/gce.ini'))
+
+            for key in config.options('gce'):
+                os.environ[key] = config.get('gce', key)
+
+            inventory = '-i inventory/gce/gce.py'
+        elif 'aws' == self.args.provider:
+            config.readfp(open('inventory/aws/ec2.ini'))
+
+            for key in config.options('ec2'):
+                os.environ[key] = config.get('ec2', key)
+
+            inventory = '-i inventory/aws/ec2.py'
+        else:
+            assert False, "invalid PROVIDER {}".format(self.args.provider)
```

Comment:
I had this as an assert since the code should never be reached.  But that is just my personal preference. I can change to raise if you feel strongly.